data-model/SDP-sample-records/textual-artifact_sample1.json

```diff
@@ -0,0 +1,65 @@
+{
+  "id": "ark:/21198/adf20c1f",
+  "type": "Textual Artifact",
+  "label": "Sinai Syriac 2A, ff. 1-152v, Gospels",
+  "unit_start": {
```

Comment:
Change unit start and end to a simpler "extent" field that is an array of folio ranges. Maybe also include the parent id in each item of the array?

Reply:
Maybe these items could be more complex with nested start and end as machine-readable?

Reply:
Maybe these items could be more complex with nested start and end as machine-readable?

Reply:
done. Changed to "extent" field with array of folio ranges. Parent IDs are implied based on the relation to the parent codicological unit. There will only be a one-to-one relationship as we will create new artifacts and units for re-binding and hypothetical reconstruction

Reply:
done. Changed to "extent" field with array of folio ranges. Parent IDs are implied based on the relation to the parent codicological unit. There will only be a one-to-one relationship as we will create new artifacts and units for re-binding and hypothetical reconstruction